### ISSUE TITLE
Add support for multiple user IDs in API v2 /users index requests

### ISSUE THREAD

**Issue by initvector** [vanilla/vanilla#5773]
API v1 has a [/users/multi](https://blog.vanillaforums.com/api-users-multi) endpoint that allowed retrieving information for multiple users. API v2 will implement similar functionality, but will instead do this with the standard /users index and a comma-separated list of user IDs.

Depends on #5756

This functionality also depends on support in [vanilla/garden-schema](https://github.com/vanilla/garden-schema)

**Comment by initvector** [comment]
The necessary support in [vanilla/garden-schema](https://github.com/vanilla/garden-schema) was added with [a PR that included field style support](https://github.com/vanilla/garden-schema/pull/17). One of the supported styles is "form", [which is in the CSV format](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#style-values).

**Comment by initvector** [comment]
Closing in favor of tracking on the associated PR: #5938
